### PR TITLE
Initialize

### PR DESCRIPTION
**Pull request by 2bleO**
In this Pr I:

- Initialized React app.
- Added React Redux.
- Structured application files using a "feature folder" approach and the ducks pattern for Redux files.
- Created Books and Categories components.

### CODE REVIEW

File: src/components/.github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (Emmyn5600) [inline src/components/.github/workflows/linters.yml:37]: - [ ] Kindly, check it again for the linters configuration in the Microverse template, the .github/workflows/linters,yml shouldn't be in the components folder.

- [ ] Remove unnecessary files in the public folder.

space6png

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (Emmyn5600) [inline README.md]: - [ ] Modify the Readme file with the project details information

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <Header />
+    <Switch>
+      <Route exact path="/">
+        <Books />
+      </Route>
+
+      <Route path="/categories">
+        <Categories />
+      </Route>
+    </Switch>
```
Comment (Emmyn5600) [inline src/App.js:17]: - [ ] The home route should come after all routes, means it has to be the last route.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @2bleO 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have not been followed.:heavy_check_mark:

Changes required :writing_hand: 
 
    * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.js (hunk 3 above)
Comment (2bleO) [reply src/App.js:17]: Sorry, I didn't understand this, Can you please be a bit more specific?

Comment (Emmyn5600) [reply src/App.js:17]: The root route or home route has to be the last one, like that:
      <Route path="/categories">
          <Categories /> 
      <Route exact path="/">
          <Books />
      </Route>

**Review by AshakaE — APPROVED** [review]
### Hi @2bleO  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**